gateway-service/src/user/entities/user.entity.ts

```diff
@@ -0,0 +1 @@
+export class User {}
```

Comment:
Is this blank by your intention? Or will add it later?

Reply:
nope not my intention